### DISCUSSION TITLE
Tell user why "No state history found" on more-info

### DISCUSSION THREAD

**Discussion by SeanPM5** [home-assistant/frontend#11175] [Other feature requests]
**Home Assistant release with the issue:**
0.98.3

**Description of problem:**
Don't know if this idea is feasible or worth the effort involved. But the current more-info dialogs simply say "No state history found" regardless of the reason _why_. 

This is OK but can be a little more user friendly by providing more context:

![nostatehistory](https://i.imgur.com/DLoRejh.png)

The wording on these would need to be tweaked, but just a few ideas:

"No state history because `history:` not found in configuration.yaml."
"No state history because of history exclude."
"No state history due to recorder excludes." 
etc

New users can often have trouble figuring out the history and recorder includes/excludes, and unknowingly have it misconfigured and be left wondering why it's not working. By pointing them in the right direction it can save a lot of troubleshooting and frustration. And [even experienced smart people like HA devs](https://github.com/home-assistant/home-assistant/issues/10685) can be tripped up by this briefly. So I think that this could be a helpful addition.

**Comment by spacegaier** [comment]
This PR of mine will inform the user if the history integration is not loaded: https://github.com/home-assistant/frontend/pull/7994

Does not cover excludes though.
